research_bench/scripts/arxiv_link.txt

```diff
@@ -0,0 +1,20 @@
+https://arxiv.org/abs/2402.13448
+https://arxiv.org/abs/2409.19100
```

Comment:
this txt is for cross-bench? if so, maybe put it somewhere else or delete?

Reply:
Those are used in the cross-domain generation

Reply:
Where do you want to put?

Reply:
maybe delete temporarily? These data are included in the crossbench.json, right?